.github/workflows/build.yml

```diff
@@ -30,13 +30,16 @@ jobs:
         ./build_rosbe_ci.sh ${{github.workspace}}/RosBE-CI
     - name: Install ccache
       run: sudo apt install ccache
+    - name: Set LLVM version
+      if: ${{ matrix.compiler == 'clang' }}
+      run: echo "LLVM_VERSION=12" >> $GITHUB_ENV
     - name: Install LLVM
       if: ${{ matrix.compiler == 'clang' }}
       run: |
-        echo "LLVM_VERSION=12" >> $GITHUB_ENV
         wget https://apt.llvm.org/llvm.sh
         chmod +x llvm.sh
         sudo ./llvm.sh $LLVM_VERSION
+        echo "D_CLANG_VERSION=-DCLANG_VERSION=$LLVM_VERSION" >> $GITHUB_ENV
```

Comment:
If I'm not mistaken, LLVM_VERSION is not used anywhere else than on configure step. Just set D_CLANG_VERSION there and this is it, no need to add more steps.

Maybe rename this to something more generic, like "CMAKE_EXTRA_ARGS" or something.

Reply:
> LLVM_VERSION is not used anywhere else than on configure step.

It is used on this very `Install LLVM` step: hence the current regression, since the step merge.

> Maybe rename this to something more generic, like "CMAKE_EXTRA_ARGS" or something.

Maybe, though I would prefer to keep it obvious for the time being: not having to support concatenation/... on this var.

Reply:
With the added suggestion, you only have one step and you set version once

Reply:
Right.